packages/apps/src/configs/wallets/chain-config.ts

```diff
@@ -28,11 +28,11 @@ export const chainsConfig: AppConfig['chains'] = {
     logo: EdgwareLogo,
     currencies: [
       {
-        currencyId: 1,
+        currencyId: 2,
         enabled: true,
       },
     ],
-    nativeCurrencyId: 1,
+    nativeCurrencyId: 2,
```

Comment:
let me know if you're not understanding these values. For background, the native currency will likely always be 0.